components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -0,0 +1,122 @@
+
+**Note:** No explicit configuration is required to install the Automate Infra views.
+
+Login to Chef Automate using the credentials. Once logged in, the page looks like as shown below:
+
+![Chef Automate](/images/automate/chef-automate-on-chef-infra-page.png)
+
+Click on **Infrastructure** located at the top bar of the page. Refer to the image below:
+
+![Chef Automate with Highlighted Infrastructure](/images/automate/chef-automate-infrastructure.png)
+
+Click on the **Chef Servers** located at the left panel of the page. The page looks like as shown below:
+
+![Chef Infra Server](/images/automate/chef-server-page.png)
+
+The above image is the **Chef Server** page which contains all the Chef Infra Servers connected to the Chef Automate. The Chef Infra Servers listed are the ones that are already configured and added to the Chef Automate. To create a new Infra Server click [here](https://docs.chef.io/automate/infra_server/).
+
+To add the created Chef Server to the Infrastructure, click on **Add Chef Server** button. Refer to the image below:
+
+![Add Chef Server](/images/automate/add-chef-server-button.png)
+
+Clicking on the **Add Chef Server** button opens a popup menu as shown below:
+
+![Add Chef Server Popup Menu](/images/automate/add-chef-server-popup-menu.png)
+
+Mention the correct Name, FQDN, and IP Address of the system. Click on **Add Chef Server**. The desired server gets added to the Chef Infra Server.
+
+**Note:** If the mentioned FQDN and IP Address is incorrect, the box displays an error as shown below:
+
+![Add Chef Server Popup Menu](/images/automate/add-chef-server-popup-menu-with-error.png)
+
+Click on the listed Servers to view the details of the specific **Chef Server**.
+
+## Connect to the Chef Infra Server Organization
+
+Selecting a specific added Chef Server opens the list of organizations in the Server. Refer to the image below:
+
+![Chef Infra Server Organization](/images/automate/chef-server-organization.png)
+
+The Organization in the Chef server listed are the ones which are already created in that specific server. To create a new Organization in the Server click [here](https://docs.chef.io/automate/infra_server/#set-up-the-chef-infra-server).
+
+To add the previously created organization to the Infrastructure, click on the **Add Chef Organization** button. Refer to the image below:
+
+![Add Chef Organization](/images/automate/add-chef-organization-button.png)
+
+Clicking on the **Add Chef Organization** button opens a popup menu as shown below:
+
+![Add Chef Organization Popup Menu](/images/automate/add-chef-organization-popup-menu.png)
+
+Add the Name, Projects, Admin User, and Admin Key. Always use the exact details that were provided at the time of setting up **Knife**. Copy the contents of the `~/.chef/USER.pem` file and past it into the **Admin Key** field. Once done click **Add Chef Organization**. The desired organization will be added to the Chef Server.
```

Comment:
We need to clairify this a bit, but I don't have the right words at the moment. I will update with a suggestion tomorrow.